module/input_handler.py

```diff
@@ -64,70 +51,154 @@ def convert_type(convertable, convert_type):
 
         return convertable
 
-    def handle_nonglobal_commands(self, user_word, user_command_list) -> bool:
+    def handle_select(self, select_parameters: list) -> None:
         """
-        Handles all non-global commands. Returns false if failed or
-        if a function has been executed(in this case no other commands can be executed after).
-        Returns true if another command can be executed after this one
+        Handles the select command,
+        which allows the user to set the target from the list of possible targets.
+        Seting a target allows the user to execute commands on that target.
         """
+        if not self._check_amount_parameters(select_parameters, 1):
+            return
+        if not select_parameters[0] in self.cli_controller.possible_targets:
+            print("\tInvalid target '{}'".format(select_parameters[0]))
+            return
 
-        if user_word.upper() not in self.cli_controller.current_node.keys():
-            self.print_command_not_found(user_word)
-            return False
-
-        if self.cli_controller.current_node[user_word.upper()].type == NodeType.COMMAND:
-            if not self.cli_controller.target:
-                print("\tNo target selected.")
-                return False
-            user_params = user_command_list[user_command_list.index(user_word) + 1 :]
-            command_params = self.cli_controller.current_node[user_word.upper()].parameter_list
-
-            if not self.check_amount_parameters(user_params, len(command_params)):
-                return False
+        self.cli_controller.set_target(select_parameters[0])
+        print("\tYou selected {}".format(select_parameters[0]))
 
-            # Validate if the user parameters are of the correct type.
-            # Print invalid type if type is invalid, this code evaluates all parameters.
-            correct_params = True
-            for i, param in enumerate(command_params):
+    def handle_help(self, params: list) -> None:
+        """
+        Prints general information and information about the target,
```

Comment:
Please document the parameter.

Reply:
Please use @param params when commenting the parameter.

Reply:
select_parameters is a list containing strings of parameters

This is still very vague for me. But I will approve the changes. This is your own choice of commenting.